### ISSUE TITLE
Have an easy way to implement "password reset flow"

### ISSUE THREAD

**Issue by LoicPoullain** [FoalTS/foal#655]
# Issue

Resetting passwords is something common. Maybe Foal should provide components to be able to do it with an email sending.

# Resources

https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Forgot_Password_Cheat_Sheet.md

**Comment by anonimusprogramus** [comment]
Yes, I was about to google it. Glad to have this feature. Thank you.

**Comment by ronakmaheshwari** [comment]
Can i work on it @LoicPoullain
